public/data/Rushikesh_Bhalerao.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Rushikesh Bhalerao",
+    "location": "Maharashtra, India 🇮🇳",
+    "bio": "A Software enthusiast, Front-end developer and ML Expert",
+    "portfolio": "https://rushikeshbhalerao.netlify.app/",
```

Comment:
```
"avatar": "https://github.com/<your-github-username>.png",
```
Add this line so your profile picture will appear on the card.

Reply:
Added 👌